src/Migrations/Version20200330155339.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20200330155339 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
```

Comment:
can you add a description ?

Reply:
Someone forgot to generate this migration, so I can't invent one; 
Also adding a description only to say "add a field X" when you can read it from the SQL statement is purely useless.

Reply:
Then you can remove the method, as it's the same as the parent's one